mobile/tailwind.config.cjs

```diff
@@ -1,7 +1,12 @@
 module.exports = {
   content: ["./App.{js,jsx,ts,tsx}", "./src/**/*.{js,jsx,ts,tsx}"],
   theme: {
-    extend: {}
+    extend: {
+      colors: {
+        lightPink: { 100: "#FFEDE6" },
+        brown: { 100: "#382E2B" }
```

Comment:
Aren't these our primary, secondary colors? Can you please check with out Figma file. Thanks

Reply:
all done